README.md

```diff
@@ -47,10 +47,10 @@ gulp
 ```
 
 ## Contributing
-Refer to [Contibuting Docuemnent]('./docs/CONTRIBUTING.MD) to contribute to this project.
+Refer to [Contibuting Docuemnent](./docs/CONTRIBUTING.MD) to contribute to this project.
```

Comment:
@adetolaraphael the changes you requested has been fixed. Simply approved or request for more changes and I'd merge the issue.